### PR TITLE
chore: enhance code readability

### PR DESCRIPTION
**Pull request by mcastagnetti**
### General Summary

Enhance code readability and simplifies logic.

### Test scenario
 Just try to run `metrx`, it should run smoothly

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -26,15 +26,20 @@ program
...
+    process.exit(1);
+};
+
 if (!program.url) {
     program.help();
     process.exit(1);
 } else {
     try {
-        start(program).then(output => {
+        start(program, errorHandler).then(output => {
```
Comment (arnaudvalle) [inline cli.js:39]: Can we simplify to `then(output => console.log)`?

File: js/output.js [hunk 2]
```diff
@@ -11,9 +12,11 @@ const { toReadableValue } = require('./utils');
  * @return {string} The ready to display table.
  */
 const buildTable = data => {
-    const head = [''].concat(Object.keys(data[0].metrics)).map(entry => {
-        return chalk.blue(entry);
-    });
+    const head = [''].concat(
+        RELEVANT_STATS.map(stat => {
```
Comment (arnaudvalle) [inline js/output.js]: Could also be simplified to `(stat => chalk.blue)`?

File: js/utils.js [hunk 3]
```diff
@@ -1,37 +1,41 @@
...
+const statsFunction = require('./stats');
 
 /**
- * Return the median value for an array of raw values.
+ * Convert raw ms values to readable values.
  *
- * @param  {Array}  values The array of values.
- * @return {number} The median value.
+ * @param   {number} ms The number of milliseconds.
+ * @returns {string} The redable value.
```
Comment (arnaudvalle) [inline js/utils.js]: typo: `readable`

File: js/utils.js [hunk 4]
```diff
@@ -1,37 +1,41 @@
...
-    values.sort((a, b) => a - b);
-    const pivot = Math.floor(values.length / 2);
-
-    return values.length % 2 ? values[pivot] : (values[pivot - 1] + values[pivot]) / 2;
-};
+const addMsSuffix = ms => `${Math.floor(ms)} ms`;
 
 /**
- * Calculate the average value for an array of raw values.
+ * Makes the diff between a time and the navigation start to get a useable time in ms.
```
Comment (arnaudvalle) [inline js/utils.js]: typo: `usable`

File: js/utils.js [hunk 5]
```diff
@@ -1,37 +1,41 @@
...
  *
- * @param  {Array}  values The array of values.
- * @return {number} The average value.
+ * @param  {integer} time            The time we want the metric from
```
Comment (arnaudvalle) [inline js/utils.js]: `.` 💋

File: js/utils.js [hunk 6]
```diff
@@ -45,6 +49,62 @@ const toCamelCase = value => {
...
+const toReadableValue = (key, value) => (BYTES_BASED_VALUES.includes(key) ? bytesToSize(value) : addMsSuffix(value));
+
+/**
+ * Transforms the metrics Object into a more readable object.
+ *
+ * @param  {Object} metrics The metrics Object we get from Chrome Dev Tools.
+ * @return {Object} The correctly translated metrics.
+ */
+const translateMetrics = ({ metrics }) => {
+    return metrics.reduce((obj, item) => {
```
Comment (arnaudvalle) [inline js/utils.js:68]: Nice

**Review by arnaudvalle — CHANGES_REQUESTED** [review]
A few tiny details but overall looks good 👍

File: cli.js (hunk 1 above)
Comment (mcastagnetti) [reply cli.js:39]: Doesn't seems to work :thinking:

**Pull request merged**